.github/workflows/test-linux-smacv2.yml

```diff
@@ -17,7 +17,7 @@ concurrency:
 
 jobs:
   unittests:
-    if: contains(github.event.pull_request.labels.*.name, 'Environments')
+    if: ${{ (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'Environments')) || (github.event_name == 'push') }}
```

Comment:
if you run every time you push you can avoid checking the label name too no?

Reply:
i am still working on it, i thought the push event is onlly triggered on the listed branches, therefore the label i checked for prs